vagrant-all.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -eo pipefail
+
+export KUBESPAWN_AUTOBUILD="true"
+export KUBESPAWN_REDIRECT_TRAFFIC="true"
+vagrant up
+
```

Comment:
Are we missing something here for actually launching kube-spawn clusters? Because `vagrant up` launches a VM, but it does not run a kube-spawn cluster inside the VM. If `KUBESPAWN_AUTOBUILD` is true, the script should run something like `~/build.sh` or `kube-spawn create ; kube-spawn start` AFAIK.

Reply:
It actually runs a kube-spawn cluster inside the VM. Please take a look again at Vagrantfile and setup-env script - they both consume those variables. When `KUBESPAWN_AUTOBUILD` is `true`, then `build.sh` script is running in the provisioning phase.

Reply:
Aha, I see.
Hmm, I'm not sure I like this autobuild logic being inside Vagrantfile, 3 times in a row. But if it works, I'm not against it for now.